app/build.gradle

```diff
@@ -8,8 +8,8 @@ android {
         applicationId "org.cimbar.camerafilecopy"
         minSdkVersion 21
         targetSdkVersion 30
-        versionCode 11
-        versionName "0.5.15"
+        versionCode 13
```

Comment:
We don't talk about what happened to version 12...

(ok, it's just that the default was `mode B`, and I concluded that might be a problem)